### ISSUE TITLE
Add more tab line indicator styles in editing containers

### ISSUE THREAD

**Issue by weyCC81** [mozilla/multi-account-containers#2777]
### Step to reproduce

More colors like asked in: https://github.com/mozilla/multi-account-containers/issues/1566 or https://github.com/mozilla/multi-account-containers/issues/2228 not just custom colors --> more in count

Alternative lines styles like: Solid line (straight), Dashed line, Dotted line, Dash-dot line, ...

### Actual behavior

Limited color options and icons are not visible outside the tap

### Expected behavior

Varieties of options for containers, which may also, would solve https://github.com/mozilla/multi-account-containers/issues/1037 partly (or at least give a workaround).

### Additional informations

There have been a discussion around browser limits of color count, may line style can solve this

### Provide a copy of Troubleshooting Information page (optional)

```plain text
Label: feature request
```

**Comment by rublev** [comment]
yes please omfg

**Comment by dannycolin** [comment]
Renaming to avoid duplicate feature requests with the original ticket for more colors.
